app/controllers/generic_files_controller.rb

```diff
@@ -9,11 +9,29 @@ class GenericFilesController < ApplicationController
   self.edit_form_class = FileEditForm
 
   def update_metadata
+    params["generic_file"]["nested_geo_bbox_attributes"].each do |box, value|
+      if [value["label"], value["bbox_lat_north"], value["bbox_lon_west"], value["bbox_lat_south"], value["bbox_lon_east"]].none? { |f| f.empty? }
```

Comment:
I think you would want _f.blank?_ here to handle values that could be empty or nil, in case one or more of the fields are not posted in the form.